### ISSUE TITLE
Autorun CPC+ (basic prompt)

### ISSUE THREAD

**Issue by FloLore** [ColinPitrat/caprice32#124]
Hi Colin,

Just to indicate that the command line which allows to launch a line of code in autorun no longer works for the configuration "CPC PLUS", previously you had allowed the emulator to recognize this command (cap32.exe -a CPC_F1 -a "  cat") as autorun. To pass the menu "F1 or F2", it works well, but nothing is written at the basic prompt.

Best regards,

FloLore

**Comment by ColinPitrat** [comment]
This is due to an effort to make the delay between events generated by autocmd predictible and faster when possible, cf pull request #108 
You could get the same result by adding more spaces before the cat (13 spaces should work):

```
cap32.exe -a CPC_F1 -a "             cat"
```

But this is not a great solution, so I also added a CAP32_DELAY keyword that allow to do what you want (will not be available in 4.5.0 but will be in latest):

```
./cap32 -a CPC_F1 -a CAP32_DELAY -a cat
```

Or less readable but works too:

```
./cap32 -a CPC_F1CAP32_DELAYcat
```

**Comment by FloLore** [comment]
Excelent, it works very well!
